app/src/main/java/chat/rocket/android/settings/presentation/SettingsView.kt

```diff
@@ -0,0 +1,3 @@
+package chat.rocket.android.settings.presentation
+
+interface SettingsView
```

Comment:
Why adding it? This interface contains no function, so it is useless.

Reply:
Added the interface should the implemented view go through any changes or the interface itself needs to have specific functions in the future.